projects/plugins/boost/tests/e2e/specs/base/get-started.test.ts

```diff
@@ -3,19 +3,18 @@ import playwrightConfig from '_jetpack-e2e-commons/playwright.config.mjs';
 import { boostPrerequisitesBuilder } from '../../lib/env/prerequisites.js';
 import { JetpackBoostPage } from '../../lib/pages/index.js';
 
-let jetpackBoostPage;
-
 test.describe( 'Getting started page', () => {
+	let page;
+	let jetpackBoostPage;
+
 	test.beforeEach( async ( { browser } ) => {
-		const page = await browser.newPage( playwrightConfig.use );
+		page = await browser.newPage( playwrightConfig.use );
 		await boostPrerequisitesBuilder( page ).withCleanEnv().withConnection( false ).build();
 
 		jetpackBoostPage = await JetpackBoostPage.visit( page );
 	} );
 
-	test.afterAll( async ( { browser } ) => {
-		const page = await browser.newPage();
-		await boostPrerequisitesBuilder( page ).withCleanEnv().withConnection( true ).build();
+	test.afterEach( async () => {
 		await page.close();
```

Comment:
This is after all. Meaning, follow up tests should make sure they open the page with a connection. Otherwise things will break.

Considering we need to open a new page for each tests, if we need to test the actual features themselves, we'd open a page with a connection. I think this makes sense.